nbs/core.ipynb

```diff
@@ -339,6 +339,8 @@
     "        self.id_col = id_col\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #96.            err_msg = ""
</pre><p>I think you could accumulate the columns with missing values instead of the error message, e.g. <code>cols_with_nans = []</code> and check at the end</p><pre class="ql-syntax" spellcheck="false">if cols_with_nans:
    raise ValueError(f"Found missing values in {cols_with_nans}.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>